src/main/resources/hibernate.cfg.xml

```diff
@@ -7,7 +7,7 @@
         <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
         <property name="hibernate.hbm2ddl.auto">create-drop</property>
         <property name="hibernate.generate_statistics">false</property>
```

Comment:
you can omit this if it is false

Reply:
I use it sometimes, but in this case, I'll remove it.